src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/utils/SessionTemplateSQLGeneratorApplication.kt

```diff
@@ -276,9 +278,10 @@ class SessionTemplateSQLGenerator {
       endDate = sessionRecord.get(END_DATE.name)?.let { LocalDate.parse(it) },
       dayOfWeek = DayOfWeek.valueOf(sessionRecord.get(DAY_OF_WEEK.name).uppercase()),
       locationKeys = sessionRecord.get(LOCATION_KEYS.name)?.uppercase(),
-      biWeekly = sessionRecord.get(BI_WEEKLY.name)?.uppercase().toBoolean(),
+      weeklyFrequency = Integer.parseInt(sessionRecord.get(WEEKLY_FREQUENCY.name)),
```

Comment:
you could check that this is greater than zero,..

Reply:
added validation